provider/spark_test.go

```diff
@@ -16,6 +16,58 @@ import (
 	"time"
 )
 
+func testMaterializeResource(store *SparkOfflineStore) error {
+	exampleStructArray := make([]exampleStruct, 10)
+	for i := 0; i < 5; i++ {
+		exampleStructArray[i] = exampleStruct{
+			Name:       fmt.Sprintf("John Smith_%d", i),
+			Age:        30 + i,
+			Score:      100.4 + float32(i),
+			Winner:     false,
+			Registered: time.UnixMilli(int64(i)),
+		}
+	}
+	for i := 0; i < 5; i++ {
```

Comment:
We can either combine the two for loops into one. Or make the second for-loop start from 5 and go up to 10. `for i := 5; i < 10; i++`

Reply:
changed